packages/block-library/src/table/save.js

```diff
@@ -64,11 +68,18 @@ export default function save( { attributes } ) {
 
 	return (
 		<figure>
-			<table className={ classes }>
+			<table className={ classes } aria-labelledby={ hasCaption ? captionId : undefined }>
```

Comment:
A comment here would be good I think. :)

Reply:
Done 👍